tests/gnu_ar.rs

```diff
@@ -0,0 +1,171 @@
+use ar::{Archive, Header, GnuBuilder, GnuSymbolTableFormat};
+use assert_cmd::Command;
+use indoc::indoc;
+use pretty_assertions::assert_eq;
+use std::{
+    collections::BTreeMap,
+    env,
+    fs::File,
+    io::Cursor,
+    path::PathBuf
+};
+use itertools::Itertools;
+use anyhow::{Result, anyhow};
+
+#[inline]
+fn as_str(raw: &[u8]) -> String {
+    String::from_utf8_lossy(raw).to_string()
+}
+
+/// Dumb naive function that returns the command if it exists and can pass some basic checks
+fn command(name: &str) -> Option<Command> {
+    match Command::new(name).arg("--version").output() {
+        Ok(out) => {
+            if out.status.success() && as_str(&out.stdout).contains("GNU") {
+                let mut command = Command::new(name);
+                command.env("TZ", "UTC");
+                Some(command)
+            } else {
+                println!(
+                    "None GNU tool ?\n{}{}",
+                    as_str(&out.stdout),
+                    as_str(&out.stderr)
+                );
+                None
+            }
+        }
+        Err(e) => {
+            println!(
+                "Command {} not found due to `{:?}` skipping test",
+                name, e
+            );
+            None
+        }
+    }
+}
+
+fn fixture_path(name: &str) -> PathBuf {
+    let root_dir =
+        env::var("CARGO_MANIFEST_DIR").expect("$CARGO_MANIFEST_DIR");
```

Comment:
```suggestion
    let root_dir = env!("CARGO_MANIFEST_DIR");
```

This way running the test directly also works.